CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Ability to choose remote video sources in `AllHighestVideoBandwidthPolicy`
```

Comment:
I'd link to the guide here.

Reply:
I have currently linked to https://aws.github.io/amazon-chime-sdk-js/modules/videolayout.html#downlink-policy. Will work with you to update that section of the document.